app/grandchallenge/workstation_configs/models.py

```diff
@@ -73,6 +73,56 @@
     },
 }
 
+KEY_BINDINGS_SCHEMA = {
+    "$schema": "http://json-schema.org/draft-06/schema",
+    "$id": "http://example.com/example.json",
+    "type": "array",
+    "title": "The Key Bindings Schema",
+    "description": "Define the key bindings for the workstation.",
+    "items": {
+        "$id": "#/items",
+        "type": "object",
+        "title": "The Key Binding Schema",
+        "description": "Defines a key binding for a command.",
+        "default": {},
+        "examples": [
+            {
+                "key": "ctrl+shift+k",
+                "command": "editor.action.deleteLines",
+                "when": "editorTextFocus",
+            }
+        ],
+        "required": ["key", "command"],
+        "additionalProperties": False,
+        "properties": {
+            "key": {
+                "$id": "#/items/properties/key",
+                "type": "string",
+                "title": "The Key Schema",
+                "description": "The keys used for this binding.",
+                "default": "",
+                "examples": ["ctrl+shift+k"],
+            },
+            "command": {
+                "$id": "#/items/properties/command",
+                "type": "string",
+                "title": "The Command Schema",
+                "description": "The command called by this binding.",
+                "default": "",
+                "examples": ["editor.action.deleteLines"],
```

Comment:
We use hyphons as separators now but well... not really important. Example:
https://github.com/DIAGNijmegen/rse-cirrus-core/blob/e89c0542b3e9b79bb64c0843547b9d415cc21585/src/cirrus/DIAG/CIRRUSCore/Modules/Macros/ToolboxPlugin/ToolboxPlugin.py#L157

Reply:
That's ok, the values are not checked, example was from the VSCode documentation.